docs/LoadTests.md

```diff
@@ -22,7 +22,9 @@ You can configure the parameters by editing `run.sh` and add/change parameters(e
 The Script works in Steps
 - Starts by creating `n` number of UserSignup CRD's which will create `n` number of NameSpaces , number of users can be changed by the flag `--users`
 - Then it proceeds by creating AppStudio Applications for each user followed by Appstudio Component, i.e Creates users on a 1:1 basis 
-- Creating the Component will start the pipelines, if the `-w` flag is given it will wait for the pipelines to finish then print results 
+- Creating the Component will start the pipelines, if the `-w` flag is given it will wait for the pipelines to finish,
+- if the '-i' flag is given it will wait till integration tests have finished  
```

Comment:
This `-i` option is not yet implemented.

Reply:
right

Reply:
removing it